### ISSUE TITLE
Please publish a new version to Hackage

### ISSUE THREAD

**Issue by domMayhew** [lucasdicioccio/http2-client#90]
I am writing [a library](https://github.com/utxorpc/spec) that depends on this package.

Please publish a new version of `http2-client` to Hackage. If that's not possible, please update the dependencies listed on Hackage to prevent the build breaking on `mtl` versions >= 2.3. It seems that many of the dependencies in the Hackage metadata for this package have inadequate bounds, allowing the use of a major version greater than specified. (E.g., `time >=1.8 && <2` should be `time >= 1.8 && < 1.9`).

Since I am writing a library and not and end-use project, depending on commits in this GitHub repo is not a viable solution.

Thanks in advance!

**Comment by lucasdicioccio** [comment]
Hi!, I'll try to come up with some updates in the upcoming week/week-end. Sorry for being a bit slow, life makes it a bit hard to get some spare-time budget these days.

**Comment by domMayhew** [comment]
Thank you! Let me know if I can help with PRs or anything!

**Comment by lucasdicioccio** [comment]
@domMayhew I've edited the 0.10.0.0 revision package bounds with your two suggested changes

I see that #89 has some fixes for warnings and some package bound checks. I'm considering merging that after some test and doing another version. Stay tuned.

**Comment by domMayhew** [comment]
Awesome, thanks! Looking forward to your PR merging and being published. Thanks for working on this!

**Comment by lucasdicioccio** [comment]
hi @domMayhew  I've published 0.10.0.1 with increased compatibility over dependent packages (and ghc-version) https://hackage.haskell.org/package/http2-client

**Comment by domMayhew** [comment]
Thanks @lucasdicioccio !

I've encountered one problem: the lower bounds on `tls` is not compatible with the upper bounds on `tls` in `http2-client-grpc`. I've tested using `tls-1.8.0` (with `stack build` and `stack test`) and it was successful. Not sure if there are other tests that need to be run though.

Is it possible to publish a meta-revision with relaxed upper bounds? I've made a PR (#93) with the one-line change.

Thank you!
